internal/provider/types_streaming_namespace_policies.go

```diff
@@ -195,6 +203,16 @@ var (
 					policyInactiveTopicMaxInactiveDurationSeconds: int64PulsarNamespacePolicyAttribute,
 				},
 			},
+			policySubscriptionExpirationPolicy: schema.SingleNestedAttribute{
+				Optional: true,
+				Computed: true,
+				PlanModifiers: []planmodifier.Object{
+					objectplanmodifier.UseStateForUnknown(),
+				},
+				Attributes: map[string]schema.Attribute{
+					policySubscriptionExpirationTime: int64PulsarNamespacePolicyAttribute,
+				},
+			},
```

Comment:
This doesn't need to be a nested attribute since we're only setting a single value.
```suggestion
			subscriptionExpirationTimeMinutes: int64PulsarNamespacePolicyAttribute
```